AllReadyApp/Web-App/AllReady/Models/Campaign.cs

```diff
@@ -71,7 +71,7 @@ public class Campaign
 
         public Location Location { get; set; }
 
-        public List<CampaignContact> CampaignContacts { get; set; }
+        public List<CampaignContact> CampaignContacts { get; set; } = new List<CampaignContact>();

```

Comment:
This is what fixed part of the issue. There is a query that is getting the task the user is volunteering, including the campaign, and then in memory getting the campaign's first contact's email addresses. However, the in-memory component had null for the CampaignContact, thus throwing a NullReference.